languages/javascript.mdx

```diff
@@ -1,19 +1,88 @@
 ---
-title: 'JavaScript'
-icon: 'square-js'
+title: "JavaScript"
+icon: "square-js"
 ---
 
+To get started, create an account by clicking “_Log in_” on [PromptLayer](https://promptlayer.com/). Once logged in, click the button to create an API key and save this in a secure location ([Guide to Using Env Vars](https://nodejs.dev/en/learn/how-to-read-environment-variables-from-nodejs/)).
 
-<Warning> Javascript support is experimental and will not be updated as frequently as the Python one. Follow [@promptlayer](https://twitter.com/promptlayer) for updates and check back here for Javascript requirements.</Warning>
+Once you have that all set up, install PromptLayer using `npm`.
 
-## LangChain for Javascript
+```bash
+npm install promptlayer
+```
 
-PromptLayer is available through LangChain’s Javascript library: [https://github.com/hwchase17/langchainjs](https://github.com/hwchase17/langchainjs)
+PromptLayer JavaScript library has support for OpenAI.
 
-Learn more at: [https://js.langchain.com/docs/modules/models/llms/integrations#promptlayeropenai](https://js.langchain.com/docs/modules/models/llms/integrations#promptlayeropenai)
+## OpenAI
 
-## Using without LangChain
+In the JavaScript file where you use OpenAI APIs, add the following. This allows us to keep track of your requests without needing any other code changes.
```

Comment:
"In the JavaScript file where the OpenAI APIs are integrated, include the following lines. They enable PromptLayer to track your requests without additional code modifications."